WebRoot/css/zm.css

```diff
@@ -2899,121 +2925,114 @@ td.calendar_month_day_item table td {
 /*
  * appearance of appointments in day views -- NOTE:colors of these items are not skinned
  */
-.appt							{ 	z-index:10;	}
-.appt-selected					{ 	z-index:20; outline: none; }
+.appt { 
+	@Appt@
+}
 
-.appt-selected TD		 		{	font-weight:bold;   }
+.appt-selected {
+	@ApptSelected@
+}
 
 /* ??? USE? */
 .appt_new_tag					{	font-weight:bold;	color:#FC0;	}
 
-.appt_name						{	@SmallBoxPadding@ font-weight:bold;  }
-.appt_new_name					{	@SmallBoxPadding@	}
+.appt_name,
+.appt_new_name {
+	@ApptName@
+}
 
 .appt_30_name,
-.appt_allday_name				{	@SmallBoxPadding@ padding-top:1px; @NoWrap@	}
 .appt_30_new_name,
-.appt_allday_new_name 			{	@SmallBoxPadding@ padding-top:1px; @NoWrap@	}
+.appt_allday_name,
+.appt_allday_new_name {
+	@ApptAllDayName@
+}
+
+.appt_time {
+	@ApptTime@
+}
+
+.appt_new_time {
+	@ApptTime@
+}
 
+.appt_end_time {
+	@ApptEndTime@
+}
 
-.appt_time						{	@SmallBoxPadding@	}
-.appt_new_time					{	@SmallBoxPadding@	}
-.appt_end_time					{	@SmallBoxPadding@ color:#666; @NoWrap@	}
-.appt_location					{	@SmallBoxPadding@ color:#666; font-weight:normal;	}
-.appt_icon						{ 	position:relative; float:right;		}
+.appt_location {
+	@ApptLocation@
+}
+
+.appt_icon {
+	@ApptIcon@
+}
 
 .appt_top_sash,
 .appt_bottom_sash{
-	height:5px;
-	width:100%;
-	position:absolute;
-	font-size:2pt;
+	@ApptSash@
 }
 
 .appt_top_sash {
-	top:0px;
-	cursor:n-resize;
+	@ApptTopSash@
 }
 .appt_bottom_sash {
-	bottom:0px;
-	cursor:s-resize;
+	@ApptBottomSash@
 }
 
 .appt_body,
 .appt_30_body,
 .appt_allday_body {
-	position:relative;
-	overflow:hidden;
-	border-style:solid;
-	border-color:#A3A3A3;
-	border-width:0px;
-	@SmallRoundCorners@
-#IFDEF GECKO
-	-moz-user-select:none;
-#ENDIF
-	margin:0px 0px 0px -1px;
+	@ApptBody@;
+	#IFDEF GECKO
+		-moz-user-select:none;
+	#ENDIF
 }
 
-.appt_body TABLE {
-	background-color:#E5E5E5;
-	border-collapse:separate;
+.appt_30_body TABLE,
+.appt_allday_body TABLE,
+.appt_body > TABLE {
+	@ApptBodyTable@
 }
 
-.appt_30_body TABLE,
-.appt_allday_body TABLE {
-	
+HTML>BODY .appt-selected .appt_body TD.appt_30_name,
+HTML>BODY .appt-selected .appt_allday_body TD.appt_30_name,
+HTML>BODY .appt-selected .appt_30_body TD.appt_30_name {
+	#IFDEF GECKO
+		-moz-user-select:none;
```

Comment:
i guess there is already a property to handle this in skin.properties

Reply:
Noted and updated here: https://github.com/Zimbra/zm-web-client/pull/145/commits/1ba1c92bb0a7e585b2f06c03988eae7ccf0a1f84

Thanks